apps/web/src/pages/SearchRequests/SearchPage/components/AdvancedFilters.tsx

```diff
@@ -0,0 +1,312 @@
+
+interface AnimatedContentProps
+  extends React.ComponentPropsWithoutRef<typeof Accordion.Content> {
+  isOpen: boolean;
+}
+
+const AnimatedContent = React.forwardRef<
+  React.ElementRef<typeof Accordion.Content>,
+  AnimatedContentProps
+>(({ isOpen, children, ...rest }, forwardedRef) => (
+  <Accordion.Content asChild forceMount ref={forwardedRef} {...rest}>
+    <motion.div
+      className="Accordion__Content"
+      animate={isOpen ? "open" : "closed"}
+      variants={animationVariants}
+      transition={{ duration: 0.2, type: "tween" }}
+    >
+      {children}
+    </motion.div>
+  </Accordion.Content>
+));
+
+type AdvancedFilter =
+  | "educationRequirement"
+  | "employmentDuration"
+  | "operationalRequirements";
+
+type AdvancedFilterOptions = Array<AdvancedFilter>;
+
+const AdvancedFilters = () => {
+  const intl = useIntl();
+  const [currentAdvancedFilters, setCurrentAdvancedFilters] =
+    React.useState<AdvancedFilterOptions>([]);
+  const { watch } = useFormContext();
+  const [educationRequirement, employmentDuration, operationalRequirements] =
+    watch([
+      "educationRequirement",
+      "employmentDuration",
+      "operationalRequirements",
+    ]);
+
+  const educationRequirementOptions = [
+    {
+      value: "no_diploma",
+      label: intl.formatMessage({
+        defaultMessage:
+          "Can accept a combination of work experience and education",
+        id: "74WtLG",
+        description:
+          "Radio group option for education requirement filter in search form.",
+      }),
+    },
+    {
+      value: "has_diploma",
+      label: intl.formatMessage({
+        defaultMessage: "Required diploma from post-secondary institution",
+        id: "KoPFx4",
+        description:
+          "Radio group option for education requirement filter in search form.",
+      }),
+    },
+  ];
+
+  const employmentDurationOptions = [
+    {
+      value: NullSelection,
+      label: intl.formatMessage({
+        defaultMessage:
+          "Any duration (short term, long term or indeterminate) (Recommended)",
+        id: "8fQWTc",
+        description: "No preference for employment duration - will accept any",
+      }),
+    },
+    {
+      value: EmploymentDuration.Term,
+      label: intl.formatMessage(getEmploymentDuration(EmploymentDuration.Term)),
+    },
+    {
+      value: EmploymentDuration.Indeterminate,
+      label: intl.formatMessage(
+        getEmploymentDuration(EmploymentDuration.Indeterminate),
+      ),
+    },
+  ];
+
+  const operationalRequirementOptions = OperationalRequirementV2.map(
+    (value) => ({
+      value,
+      label: intl.formatMessage(getOperationalRequirement(value)),
+    }),
+  );
+
+  const operationalRequirementOptionsShort = OperationalRequirementV2.map(
+    (value) => ({
+      value,
+      label: intl.formatMessage(getOperationalRequirement(value, "short")),
+    }),
+  );
+
+  return (
+    <>
+      <Heading level="h3" size="h5" data-h2-font-weight="base(700)">
```

Comment:
It looks a little odd that this is bigger than the other headers.
![image](https://user-images.githubusercontent.com/8978655/216635794-e4f90888-473d-4374-bdfb-70e4ab34b63d.png)
On my machine it is 23.04 px vs 19.02 px.

Reply:
I'll make this change but I think we may need to have designers look over the size of headers here. It is odd that all headers are the size of an `h6` regardless of rank. We should have a visual difference in heading rank to communicate it to sighted users.